drud/ddev
# Implementation of global project-tld config option

## The Problem/Issue/Bug:
I want to set a global project tld by default to support a custom TLD for out company.

## How this PR Solves The Problem:
Adds a config global config option for project-tld that overwrites the default for the project and makes it globally settable.

## Manual Testing Instructions:
Build, use --project-tld globally and see it work. You can also set it in the global config yaml.

## Automated Testing Overview:
Added the option in the tests, but can't run tests locally, so thats a little bit hard.

I did test this manually by setting the global config in yml and observing our actual project tld be used! Hopefully i can use gitpod to run the tests.

## Related Issue Link(s):
https://github.com/drud/ddev/issues/2746

## Release/Deployment notes:
